ISISComputingGroup/EPICS-ioc
# Ticket1843: Nanodac Support

### Description of work

Added the nanodac IOC.

### To test

You will also need https://github.com/ISISComputingGroup/EPICS-nanodac

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1843

Add the nanodac as part of a configuration. It is currently on 130.246.49.95. 
Check that you can read all four channels on %MYPVPREFIX%NANODAC:CH1:PV etc. PVs should be around room temperature. 

---

#### Code Review


### Functional Tests


### Final steps
